README.md

```diff
@@ -26,10 +26,11 @@ node seeds/index.js
 ## Versiones
 
 1.0 - Feat: Agrega esqueleto base backend.
+2.0 - Feat: Agrega rutas, controllers, middlewares, logger, control básico de errores, validación básica.
 
 ## Mapa de Ruta
 
-El proyecto se considerará terminado una vez entregado con todas las correcciones pertinentes. En un futuro se espera incluir reservaciones y calendarización.
+El proyecto se considerará terminado una vez entregado con todas las correcciones pertinentes. En un futuro se espera incluir reservaciones, reseñas y otras funcionalidades.
 
 ## Contact
```

Comment:
It will be nice to have a way to contact you; besides your `GitHub` profile should be nice to have your portfolio link, a link to your **Linked In** profile, here is the best place to have contact with recruiters, so I belive that this will give you a better impact to have your accounts connected!. 🤩😱🚀